_posts/2020-02-21-release-v0-3-13.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title:  "Release v0.3.11"
```

Comment:
Should 0.3.13 be here instead of 0.3.11?

Reply:
Indeed it should!